packages/cli/commands/start.ts

```diff
@@ -211,20 +214,43 @@ export class Start extends Command {
 				// Wait till the database is ready
 				await startDbInitPromise;
 
+				const encryptionKey = await UserSettings.getEncryptionKey();
+
+				if (!encryptionKey) {
+					throw new Error(RESPONSE_ERROR_MESSAGES.NO_ENCRYPTION_KEY);
+				}
+
 				if (config.get('userManagement.emails.mode') === 'smtp') {
-					const smtpConfiguration = config.get('userManagement.emails.smtp');
+					const { auth, ...rest } = config.get('userManagement.emails.smtp');
+
+					const encryptedAuth = {
+						user: auth.user,
+						pass: AES.encrypt(auth.pass, encryptionKey).toString(),
+					};
+
 					await Db.collections.Settings!.save({
 						key: 'userManagement.emails.smtp',
-						value: JSON.stringify(smtpConfiguration),
+						value: JSON.stringify({ ...rest, auth: encryptedAuth }),
 						loadOnStartup: false,
 					});
 				} else {
 					// If we don't have SMTP settings, try loading from db.
-					const smtpSettings = await Db.collections.Settings!.findOne({
+					const smtpSetting = await Db.collections.Settings!.findOne({
 						key: 'userManagement.emails.smtp',
 					});
-					if (smtpSettings) {
-						config.set('userManagement.emails.smtp', JSON.parse(smtpSettings.value));
+
+					if (smtpSetting) {
+						const { auth, ...rest } = JSON.parse(smtpSetting.value) as SmtpConfig;
+
+						const decryptedAuth = {
+							user: auth.user,
+							pass: AES.decrypt(auth.pass, encryptionKey).toString(enc.Utf8),
+						};
+
+						console.log({ ...rest, auth: decryptedAuth });
+
+						config.set('userManagement.emails.mode', 'smtp');
```

Comment:
Added this line assuming that if they have an SMTP config they will want to use it. Let me know otherwise.